### DISCUSSION TITLE
svelte build error

### DISCUSSION THREAD

**Discussion by Mowitra** [Mair/esp32-course#150] [Course Qeustions]
website on chip >site in SPIFFS .
In the above mentioned video  i followed every steps as you told but i got the following error while npm run build..
please check and reply sir.

PS C:\ESP_IDF_PROJECTS\example_site_1\site> npm run dev
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> site@0.0.1 dev
> vite dev

  VITE v3.0.2  ready in 1261 ms

  ➜  Local:   http://127.0.0.1:5173/
  ➜  Network: use --host to expose

🌼 daisyUI components 2.19.1  https://github.com/saadeghi/daisyui
  ✔︎ Including:  base, components, themes[29], utilities
  
Terminate batch job (Y/N)? y
PS C:\ESP_IDF_PROJECTS\example_site_1\site> npm run build
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> site@0.0.1 build
> vite build

vite v3.0.2 building for production...
transforming (6) .svelte-kit\generated\root.svelte
🌼 daisyUI components 2.19.1  https://github.com/saadeghi/daisyui
  ✔︎ Including:  base, components, themes[29], utilities
  
✓ 14 modules transformed.
.svelte-kit/output/client/manifest.json                                      1.37 KiB
.svelte-kit/output/client/_app/immutable/start-a2c8b7d1.js                   23.95 KiB / gzip: 9.00 KiB
.svelte-kit/output/client/_app/immutable/pages/__layout.svelte-c0a8980a.js   0.53 KiB / gzip: 0.35 KiB
.svelte-kit/output/client/_app/immutable/error.svelte-9dafedec.js            1.56 KiB / gzip: 0.75 KiB
.svelte-kit/output/client/_app/immutable/pages/index.svelte-f6f9f0be.js      1.65 KiB / gzip: 0.85 KiB
.svelte-kit/output/client/_app/immutable/chunks/index-44b335a2.js            6.98 KiB / gzip: 2.85 KiB
.svelte-kit/output/client/_app/immutable/assets/__layout-45ad09aa.css        31.07 KiB / gzip: 6.26 KiB
vite v3.0.2 building SSR bundle for production...
✓ 12 modules transformed.
Generated an empty chunk: "hooks"
.svelte-kit/output/server/manifest.json                             1.16 KiB
.svelte-kit/output/server/index.js                                  76.23 KiB
.svelte-kit/output/server/entries/pages/__layout.svelte.js          0.27 KiB
.svelte-kit/output/server/entries/fallbacks/error.svelte.js         0.74 KiB
.svelte-kit/output/server/entries/pages/index.svelte.js             0.81 KiB
.svelte-kit/output/server/_app/immutable/chunks/index-3e01a4ca.js   2.61 KiB
.svelte-kit/output/server/_app/immutable/chunks/hooks-bced8853.js   0.00 KiB
(node:16540) ExperimentalWarning: buffer.Blob is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[vite-plugin-svelte-kit] 404 /      https://kit.svelte.dev (linked from /)
error during build:
**Error: 404 /      https://kit.svelte.dev (linked from /)
    at file:///C:/ESP_IDF_PROJECTS/example_site_1/site/node_modules/@sveltejs/kit/dist/vite.js:1124:11
    at save (file:///C:/ESP_IDF_PROJECTS/example_site_1/site/node_modules/@sveltejs/kit/dist/vite.js:1394:4)
    at visit (file:///C:/ESP_IDF_PROJECTS/example_site_1/site/node_modules/@sveltejs/kit/dist/vite.js:1285:3)**

**Comment by Mair** [comment]
@Mowitra,
Does it work when you run it in dev?
hmmm. this is a tough one. I can't see from the output what the issue might be.
One thing I do see is you are getting a 404 (not found error) on https://kit.svelte.dev/
Are you offline?
